src/components/Security.js

```diff
@@ -9,57 +9,49 @@ import "../login/styles/index.scss";
 class Security extends Component {
   constructor(props) {
     super(props);
+
     this.state = { 
-      isAvailablePlatformAuthenticator: false,
       className: "" 
     };
-  }
 
-  componentDidMount(){
-    this.checkWebauthnDevice();
-  }
-
-  checkWebauthnDevice(){
-    if (window.PublicKeyCredential) {
-      PublicKeyCredential.isUserVerifyingPlatformAuthenticatorAvailable()
-        .then((available) => {
-          if (available) {
-            console.log("Supported."),
-            this.setState(() => {
-              return {
-                isAvailablePlatformAuthenticator: true,
-                className: "second-option"
-              };
-            });
-          } else {
-            console.log(
-              "WebAuthn supported, Platform Authenticator *not* supported."
-            ),  
-            this.setState(() => {
-              return {
-                isAvailablePlatformAuthenticator: false,
-                className: "btn-primary"
-              };
-            });
-          }
-        })
-        .catch((err) => console.log(err, "Something went wrong."));
-     } else {
-      console.log("Not supported."),
-      this.setState(() => {
-        return {
-          isAvailablePlatformAuthenticator: false,
-          className: "btn-primary"
-        };
-      });
-     }
-  }
+  let platform = false;
```

Comment:
is platform used anywhere else except for inside the `if (window.PublicKeyCredential) {` clause?

Reply:
its only used in if clause